cmd/feed-dns/main.go

```diff
@@ -104,4 +116,9 @@ func validateConfig() {
 		log.Error("Must supply r53-hosted-zone")
 		os.Exit(-1)
 	}
+
+	if (internalIngressFQDN != "") != (externalIngressFQDN != "") { //XOR
```

Comment:
We should probably also have a check for argument exclusivity - given that if these parameters are present we will ignore the elb label argument, we should fail to run if both are specified.